app/assets/javascripts/analytics/_events.js

```diff
@@ -88,13 +88,31 @@
 
   GOVUK.GDM.analytics.events = {
     'supplierListDownload': function (e) {
-      var linkClick = new LinkClick(e);
+      var currentURL = GOVUK.GDM.analytics.location.pathname();
 
-      if ((linkClick.category() === 'download') && (linkClick.fileType() === 'csv')) {
-        GOVUK.analytics.trackEvent('download', 'csv', {
-          'label': downloadLinkLabel(linkClick),
-          'transport': 'beacon'
-        });
+      // check we're on the right page
+      if (currentURL.match('\/buyers\/frameworks\/([a-z\-]+)\/requirements\/([a-z\-]+)\/*(\d+)*')) {
```

Comment:
I'm not sure this regex will work e.g. `([a-z\-]+)` will not match `digital-outcomes-and-specialists-2`.

Reply:
And with a long regex like this it would be good to have a comment next to it that explains exactly what the regex is supposed to match, so that anyone looking at it later doesn't have to figure it out for themselves.

e.g. 
```
// check we're on a "start" page for building requirements
// eg. /buyers/frameworks/digital-outcomes-and-specialists-2/requirements/user-research-participants
...
```

Actually, is that the thing this is supposed to track?
The regex you have has digits at the end... is it the csv of brief responses that suppliers download?

Reply:
We should double check exactly what `supplierListDownload` means.  Unless you're sure?

Reply:
Also, it can be a lot clearer if you use raw Python strings when working with regex patterns, which stop you having to escape things.

`'\/buyers\/frameworks\/([a-z\-]+)\/requirements\/([a-z\-]+)\/*(\d+)*'`
can be
`r'/buyers/frameworks/([a-z-]+)/requirements/([a-z-]+)/*(\d+)*'`

But as Kev suggests, this regex looks malformed towards the end. Maybe we can go over it?

Reply:
Thanks for spotting this, we just fixed it now with Kat :)